packages/components/form-field/src/FormFieldProvider.tsx

```diff
@@ -52,4 +52,10 @@ export const FormFieldProvider = ({
   return <FormFieldContext.Provider value={value}>{children}</FormFieldContext.Provider>
 }
 
+const utils = {
```

Comment:
Not sure to understand why switching to this kind of pattern... :thinking:

Reply:
Yep, it doesn't convince me too much... Couldn't be defined as a function in the provider file scope?